flybywiresim/a32nx
# feat: add custom font file for RTPI

Fixes #[issue_no]

## Summary of Changes

Added a separate font file based on @tyler58546 's custom RMP font, with new 'L' and 'R' characters, to replace the missing RTPI font file.

## Screenshots (if necessary)


**BEFORE:**
![image](https://user-images.githubusercontent.com/29005005/120651859-d99fc180-c49c-11eb-8a78-74791bb42f92.png)

**AFTER:**
![image](https://user-images.githubusercontent.com/29005005/120665077-15d91f00-c4a9-11eb-97da-fad53b27c03b.png)
![image](https://user-images.githubusercontent.com/29005005/120664832-e1fdf980-c4a8-11eb-8e9f-b80d863179c3.png)



## References

https://www.youtube.com/watch?v=L5W1HgJIahU




## Additional context



Discord username (if different from GitHub): sidnov#8337

## Testing instructions


1. Check the rudder trim position indicator (RTPI) on runway/cold and dark
2. Try max deflections on both L and R sides and ensure there is no overlap or missing characters
3. Check if the functionality is unaffected (pressing reset once should be enough to reset to 0, rudder pedal animations should work normal, etc.)


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
